### PR TITLE
New AHU and MAU types

### PR DESCRIPTION
**Pull request by evgeniya-engel**

### CODE REVIEW

**Comment by cstirdivant** [comment]
Hey @evgeniya-engel thanks for opening! Next time, please tag me or request me as a reviewer so I am notified when you open a PR

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -390,10 +392,24 @@ EDPM:
   guid: "6051d016-4a78-42f4-aeaf-3701417b8773"
   description: "Exhaust air damper percentage monitoring."
   is_abstract: true
+  opt_uses:
+  - failed_exhaust_air_damper_alarm
   uses:
   - exhaust_air_damper_percentage_command
   - exhaust_air_damper_percentage_sensor
 
+EDPM2X:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:401]: You need to add `implements:` here

Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:401]: You need to add `implements:` here

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 2]
```diff
@@ -7051,6 +7120,7 @@ RFVSC2X:
   implements:
   - OPERATIONAL
 
+
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:7123]: nit: please remove extra line

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 3]
```diff
@@ -4403,6 +4403,108 @@ AHU_SFSS_SFVSC_EFSS_EFVSC_CHWSC_HWSC_ECON_EDBPC_SARC_SSPC_OAFMC:
...
+  - EDPM2X
+  - RD
+  - UV
+  - SHM
+
+AHU_SFSS2X_SFVSC2X_RFSS2X_RFVSC2X_LHC_CHW2XSC_DH2XSDPC_SSPC_SARC_MOAFC_ECON_EDPM_RD_UV_SHM:
+  guid: "136897bb-8b58-47cb-aeff-4d1a9518dcc8"
+  description: "Multi-zone AHU with dual supply and return fans, dual chilled water coil cooling and dehumidification control, leaving temperature heating water valve control, exhaust damper monitoring, minimum ventilation control, return damper flow control, ultraviolet lamp and supply air humidity monitoring."
+  is_canonical: true
+  implements:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:4453]: doesn't have `AHU`?

File: ontology/yaml/resources/HVAC/entity_types/AHU.yaml [hunk 4]
```diff
@@ -4403,6 +4403,108 @@ AHU_SFSS_SFVSC_EFSS_EFVSC_CHWSC_HWSC_ECON_EDBPC_SARC_SSPC_OAFMC:
...
+  - ECON
+  - EDPM
+  - RD
+  - UV
+
+AHU_SFSS2X_SFVSC2X_RFSS2X_RFVSC2X_HWSC_CHWSC_SSPC_MOAFC_ECON_EDPM_RD_UV:
+  guid: "09017cb5-0a96-49e6-ba9a-b4a7004eee1f"
+  description: "Multi-zone AHU with dual supply and return fans, chilled and heating water valve control, exhaust damper monitoring, minimum ventilation control, return damper flow control and ultraviolet lamp."
+  is_canonical: true
+  implements:
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/AHU.yaml:4494]: Missing `AHU`

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 5]
```diff
@@ -1441,6 +1472,21 @@ RHDHC:
   implements:
   - CONTROL
 
+DH2XSDPC:
+  guid: "2e83d32e-360e-4ac2-afa4-5adb333ded0b"
+  description: "Supply air dewpoint temperature dehumidification control on dual cooling coil."
+  is_abstract: true
+  opt_uses:
+  - supply_air_relative_humidity_sensor
+  - failed_dehumidifier_alarm
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1481]: what would this alarm mean? you don't have a dehumidifier device, you have two coils

**Review by cstirdivant — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 5 above)
Comment (evgeniya-engel) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:1481]: good point, and we didn't have a BMS point for it, removed this alarm

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**